indico/modules/rb_new/client/js/modules/calendar/Calendar.jsx

```diff
@@ -183,15 +218,17 @@ export default connect(
     state => ({
         isFetching: calendarSelectors.isFetching(state),
         calendarData: calendarSelectors.getCalendarData(state),
+        filters: calendarSelectors.getFilters(state),
         rooms: roomsSelectors.getAllRooms(state),
+        hasFavoriteRooms: userSelectors.hasFavoriteRooms(state)
     }),
     dispatch => ({
         actions: bindActionCreators({
             fetchCalendar: calendarActions.fetchCalendar,
             fetchRoomDetails: roomsActions.fetchDetails,
             setDate: (date) => calendarActions.setDate(serializeDate(date)),
-        }, dispatch),
-        dispatch
+            setFilterParameter: (param, value) => filtersActions.setFilterParameter('calendar', param, value)
+        }, dispatch)
```

Comment:
the second dispatch there was not by mistake; it's needed for `pushStateMergeProps`.